src/sections/Art/components/ArtListItem.js

```diff
@@ -0,0 +1,48 @@
+import SubjectList from "./SubjectList";
+
+function ArtListItem({ artic }) {
+    return (
+        <>
+            {
+                artic.map((art) => {
```

Comment:
This component is called `ArtListItem` but it actually renders multiple list items. I think you should rename this component to `ArtList`, and then inside your map each of the list items should be an `ArtListItem`